cf_remote/packages.py

```diff
@@ -155,6 +160,7 @@ def init_download(self):
                 artifact = Artifact(blob)
                 artifact.add_tag(header)
                 self.artifacts.append(artifact)
+        self.artifacts = sorted(self.artifacts, key=lambda artifact: artifact.filename)
```

Comment:
@vpodzime I added this sorting line to make results predictable for testing. e.g. python 3.5 was getting a different order so the tests were failing. This will generally place older packages before newer ones and re-order based on version and distribution which we may not want to change. But generally should be "OK" as we are still matching tags and such.

Reply:
Yes. This changes "fixes" the python 3.5 test run.

Reply:
:+1:

Reply:
@craigcomstock Did you consider preserving the order instead? I guess it's just a matter of using OrderedDicts?

For example, sorting fails when numbers increase from single digit to multi digit:

```
$ python3 -c 'print(sorted(("debian9", "debian8", "debian10", "debian11")))'
['debian10', 'debian11', 'debian8', 'debian9']
```

Reply:
Yes, turns out python 3.5 json module doesn't preserve order of loaded json so needed to add `object_pairs_hook=OrderedDict` to the `json.loads()` call. :+1:

I would guess it is still possible that cf-remote will choose different packages with python3.5 but the package should still generally be "right".